__docs__/wonder-blocks-form/accessibility.stories.tsx

```diff
@@ -6,7 +6,7 @@ import {TextArea} from "@khanacademy/wonder-blocks-form";
 import {spacing} from "@khanacademy/wonder-blocks-tokens";
 
 export default {
-    title: "Packages / Form / Accessibility",
+    title: "", // Empty title so that these examples don't show in the sidebar. Examples are referenced in the .mdx file
```

Comment:
**praise:** Neat! I didn't know about this.

Reply:
Disregard this! It wasn't working as expected, the story was still there in another section if I scrolled down 😅 I found if I named these stories the same as the Overview title, it isn't shown in the sidebar